### PR TITLE
Spi connectivity check for c1101

### PR DESCRIPTION
**Pull request by crasu**
## Description:

I had messed up the wiring from the c1101 to my esp. This will do a basic connectivity check from the esp to the c1101 and print out a warning/success message.

### CODE REVIEW

File: main/ZgatewayRF.ino [hunk 1]
```diff
@@ -111,6 +111,12 @@ void setupRF() {
   Log.notice(F("RF_EMITTER_GPIO: %d " CR), RF_EMITTER_GPIO);
   Log.notice(F("RF_RECEIVER_GPIO: %d " CR), RF_RECEIVER_GPIO);
 #  ifdef ZradioCC1101 //receiving with CC1101
+  if (ELECHOUSE_cc1101.getCC1101()) {
+    Serial.println("C1101 spi Connection OK");
```
Comment (1technophile) [inline main/ZgatewayRF.ino]: Could you use `Log.notice` and `Log.error` instead, please?

Comment (crasu) [reply main/ZgatewayRF.ino]: Obsolete due to change to radiolib

**Comment by crasu** [comment]
Lib is still used for rf gateway

**Comment by 1technophile** [comment]
Thanks

**Comment by rds76** [comment]
You should use Flash strings for static Logs and end them with CR
`Log.notice(F("C1101 spi Connection OK" CR));`

**Comment by 1technophile** [comment]
Good catch

**Pull request merged**